kairos-cli/bin/commands/deposit.rs

```diff
@@ -0,0 +1,31 @@
+use crate::common::{amount, private_key};
+use crate::crypto::signer::CasperSigner;
+use crate::error::CliError;
+use clap::{ArgMatches, Command};
+
+pub struct Deposit;
```

Comment:
Please make this more declarative to reduce the cognitive overhead for finding out what arguments the deposit command expects and what the arguments types are i.e. use claps derive features https://docs.rs/clap/latest/clap/_derive/index.html
```rust
#[derive(Parser)]
pub struct Cli {
    #[clap(subcommand)]
    pub command: Command,
}
#[derive(Subcommand)]
pub enum Command {
    #[clap(subcommand)]
    Deposit {
    
    },
    Transfer {
    },
    Withdraw {
    }
}
#[derive(Subcommand)]
pub enum ProveCommand {
    Hypotenuse {
        
        a: u64,
        #[clap(short, long)]
        b: u64,
    },
    Polynomial {
        #[clap(short, long)]
        a: i64,
        #[clap(short, long)]
        x: i64,
        #[clap(short, long)]
        b: i64,
    },
}
```
In order to find out what arguments the `Deposit` command requires I have to first read `new_cmd` then I have to look for `amount::arg()` in a completely different file. In that file I then have to look up static variables to figure out their names are and in order to find the type or the argument I have to read the `get()` method.